geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/GetStackTracesFunction.java

```diff
@@ -34,8 +37,10 @@ public void execute(FunctionContext context) {
       if (memberNameOrId == null) {
         memberNameOrId = cache.getDistributedSystem().getDistributedMember().getId();
       }
+      String timestamp = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss.SS").format(new Date());
```

Comment:
Last request from my side, promise ✋: the `StackTracesPerMember` domain bean should have the full `Date` object, and the actual `Date` representation should be rendered in the "view" layer directly, so I'd move this part to the actual gfsh command as well.

Reply:
Hi @jujoramos , do you mean to move this line outside of the try/catch block or to the `ExportStackTraceCommand` before _for_ loop? Or to have `new Date()` in the `StackTracesPerMember` and formatting it in the `ExportStackTraceCommand`?

Reply:
Hey @mkevo,

I mean that the `timestamp` attribute within the `StackTracesPerMember` class should be of type `Date`, not `String`. The actual formatting should happen on the `ExportStackTraceCommand`.